src/bundle.ts

```diff
@@ -13,7 +13,14 @@ export * from './typings';
 export * from './decoration';
 export * from './package';
 
-const rpc = new RPC(window.top, '1.0');
+const rpc = new RPC(
+  window.top,
+  '1.0',
+  '*',
+  undefined,
```

Comment:
If these are undefined here can we move the arguments on the constructor so that they can be omitted?

Reply:
```

Reply:
```

Reply:
```
 private readonly target: IPostable,
    protocolVersion: string,	    protocolVersion: string,
    private readonly origin: string = '*',	    private readonly origin: string = '*',
private readonly isApp?: boolean,
    private readonly setOnMessage?: (fn: (ev: any) => void) => void,
    private readonly removeOnMessage?: (fn: (ev: any) => void) => void,
    
```

Would then mean we wouldn't have to include them?

Reply:
I've reordered these so they can be omitted 😄